src/projects/projects.service.ts

```diff
@@ -126,26 +127,25 @@ export class ProjectsService {
         throw new InternalServerErrorException(`Failed to delete project ${projectId}`);
       });
 
-      // Send the grpc request to delete the repository
-      const repositoryRequest: RepositoryRequest = createRepositoryRequest(projectId);
+      // TODO:
+      // Check if the error is a 404 meaning that the image does not exist and we can continue
```

Comment:
Has this been implemented yet? If yes, please remove the TODOs

Reply:
yep waiting for team infra to implement the right error msg to finish this part

Reply:
done :+1: